charts/devtron/templates/dex.yaml

```diff
@@ -100,7 +100,7 @@ spec:
         - -n
         - /authenticator
         - /shared
-        image: {{ .initContainer.authenticator }}
+        image: {{ include "common.image" (dict "component" $.Values.components.argocdDexServer "global" $.Values.global  "extraImage" .initContainer.authenticator) }}
```

Comment:
confirm if extraImage value is correct here and it's working, we haven't used $ here

Reply:
It was working as we are using with command in above , for now I have changed to $.Values